src/nm-service/nm-food-count-input.service.ts

```diff
@@ -318,16 +319,6 @@ Example:
             return [lbsCount, contentList.join(' ')];
         }
 
-        // in this case the number was last
```

Comment:
So this was kinda left open and undecided. I think we should leave it in.

Reply:
It seemed like everyone was in agreement that this was annoying. I want to be able to say "meet you at 8" or other things like that. Plus multiple formats make things unnecessarily confusing.